lib/entry/crossDCsStreamPersistence.js

```diff
@@ -200,7 +209,7 @@ class CrossDCsStreamPersistence {
     }
 
     toJSON() {
-        return Object.keys(this._entriesToProcess);
+        return Object.keys(this._entriesToProcess).toString();
```

Comment:
not good... since this is what is returned to the diagnostics, and we don't want it as string but as object?